app/models/session.rb

```diff
@@ -48,6 +48,7 @@ class Session < ApplicationRecord
   delegate :published?, to: :intervention
   delegate :draft?, to: :intervention
   delegate :ability_to_update_for?, to: :intervention
+  delegate :warning_screen_enabled, to: :intervention
```

Comment:
I’m wondering if we should have the option to set this at both the intervention level and the specific session level. I’m not sure how to interpret `Add option in Session and Intervention settings to remove Warning Screen` from the AC.